lib/board.rb

```diff
@@ -12,6 +12,15 @@ def initialize
       row_6: ".......",
       }
     @open_row = [6,6,6,6,6,6,6]
-    @playable_col = [0,1,2,3,4,5,6]
+    @open_col = [0,1,2,3,4,5,6]
```

Comment:
I like both the grid and open_col refactors.  Makes the code easier to write and follow.